src/labels/CustomLabels.labels

```diff
@@ -2416,6 +2416,155 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUILongDeployment</shortDescription>
+        <value>Modifying this Field Mapping is taking longer than expected.</value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUILongDeploymentLink</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUILongDeploymentLink</shortDescription>
+        <value>view the deployment status here.</value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUILongDeploymentMessage</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUILongDeploymentMessage</shortDescription>
+        <value>It'll continue to process in the background. Please wait or </value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUINewFieldMapping</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUINewFieldMapping</shortDescription>
+        <value>Create New Field Mapping</value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUINoFieldMappings</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUINoFieldMappings</shortDescription>
+        <value>There are no Field Mappings for this Mapping Group yet, </value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUISearchSourceInputLabel</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUISearchSourceInputLabel</shortDescription>
+        <value>Search for the source field by label or API name.</value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUISearchTargetInputLabel</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUISearchTargetInputLabel</shortDescription>
+        <value>Search for the target field by label or API name.</value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUISourceObject</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUISourceObject</shortDescription>
+        <value>Source (NPSP Data Import Object)</value>
+    </labels>
+    <labels>
+        <fullName>bdiFMUISuccessful</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiFMUISuccessful</shortDescription>
+        <value>successful</value>
```

Comment:
Change toast message to "Successful Field Mappings update." and "Unsuccessful Field Mappings update." @melissabarber